src/passkit.cs

```diff
@@ -350,36 +366,46 @@ interface PKPaymentRequest {
 		PKShippingType ShippingType { get; set; }
 
 		[iOS (9,0)]
-		[NullAllowed, Export ("shippingContact", ArgumentSemantic.Retain)]
+		[NullAllowed, Export ("shippingContact", ArgumentSemantic.Strong)]
 		PKContact ShippingContact { get; set; }
 
 		[iOS (9,0)]
-		[NullAllowed, Export ("billingContact", ArgumentSemantic.Retain)]
+		[NullAllowed, Export ("billingContact", ArgumentSemantic.Strong)]
 		PKContact BillingContact { get; set; }
+
+		[Watch (3,0)][iOS (10,0)]
+		[Static]
+		[Export ("availableNetworks")]
+		NSString[] AvailableNetworks { get; }
 	}
 
+	[Watch (3,0)]
 	[iOS (8,0)]
 	[BaseType (typeof (NSObject))]
 	interface PKPaymentToken {
-		[Export ("paymentInstrumentName")]
+
+		[NoWatch]
+		[Export ("paymentInstrumentName", ArgumentSemantic.Copy)]
 		[Availability (Deprecated = Platform.iOS_9_0, Message = "On iOS 9 and higher, use PaymentMethod instead")]
 		string PaymentInstrumentName { get; }
 
+		[NoWatch]
 		[Export ("paymentNetwork")]
 		[Availability (Deprecated = Platform.iOS_9_0, Message = "On iOS 9 and higher, use PaymentMethod instead")]
 		string PaymentNetwork { get; }
 
 		[Export ("transactionIdentifier")]
 		string TransactionIdentifier { get; }
 
-		[Export ("paymentData")]
+		[Export ("paymentData", ArgumentSemantic.Copy)]
 		NSData PaymentData { get; }
 
 		[iOS (9,0)]
-		[Export ("paymentMethod")]
+		[Export ("paymentMethod", ArgumentSemantic.Strong)]
 		PKPaymentMethod PaymentMethod { get; }		
 	}
 
+#if !WATCH
```

Comment:
Minimizing changes in branches makes merging easier: less conflicts, less time required, less mistakes... less headaches.